src/dll/fuse/fuse_intf.c

```diff
@@ -1006,6 +1006,16 @@ static NTSTATUS fsp_fuse_intf_Open(FSP_FILE_SYSTEM *FileSystem,
         break;
     }
 
+    if (GrantedAccess & FILE_APPEND_DATA)
```

Comment:
The value of `FILE_APPEND_DATA` means "append data" for files, but "add subdirectory" for directories.

```C
#define FILE_WRITE_DATA           ( 0x0002 )    // file & pipe
#define FILE_ADD_FILE             ( 0x0002 )    // directory

#define FILE_APPEND_DATA          ( 0x0004 )    // file
#define FILE_ADD_SUBDIRECTORY     ( 0x0004 )    // directory
```

Perhaps we should do this test for directories only?

Reply:
Ah, good point. I'll fix that.